client/app/components/expiration/service-expiration-date.component.controller.js

```diff
@@ -41,7 +41,7 @@
         }
 
         isAutoRenew () {
-            return this.serviceInfos.renew && (this.serviceInfos.renew.automatic || this.serviceInfos.renew.forced);
+            return this.serviceInfos && this.serviceInfos.renew && (this.serviceInfos.renew.automatic || this.serviceInfos.renew.forced);
```

Comment:
`_(this.serviceInfos).get("renew.automatic") || _(this.serviceInfos).get("renew.forced")`